### PR TITLE
Configurable Cassandra consistency level

### LINKED ISSUE #1013: Being able to configure the Cassandra quorum
In the configuration file. http://docs.datastax.com/en/cassandra/2.0/cassandra/dml/dml_config_consistency_c.html

This is also reflected in errors like: 
```
[lua] responses.lua:61: execute(): Cassandra returned error (Unavailable exception): Cannot achieve consistency level ONE
```

### PR DESCRIPTION
**Pull request by thefosk**
Closes #1013.

### CODE REVIEW

**Comment by thefosk** [comment]
@thibaultCha can you review this?

File: spec/unit/dao/cassandra/factory_spec.lua [hunk 1]
```diff
@@ -23,6 +23,7 @@ describe("Cassadra factory", function()
         contact_points = dao_properties.contact_points,
         keyspace = dao_properties.keyspace,
         query_options = {
+          consistency = 1,
```
Comment (thibaultCha) [inline spec/unit/dao/cassandra/factory_spec.lua]: this should use the value from the `cassandra.consistencies` enum and not the underlying value

**Comment by thefosk** [comment]
I have updated the PR. If it looks good I will squash and merge.

**Comment by thibaultCha** [comment]
Ok. Those tests don't exist in feature/postgres anyways

**Pull request merged**